tests/helpers/factories.py

```diff
@@ -97,7 +97,12 @@ def wrapper(operator, array_interface, *args, **kwargs):
         else:
             op = operator
         if array_interface in ["torch-gpu", "cupy"]:
-            if op.backend not in ["cufinufft", "gpunufft", "torchkbnufft-gpu"]:
+            if op.backend not in [
+                "cufinufft",
+                "gpunufft",
+                "torchkbnufft-gpu",
+                "tensorflow",
```

Comment:
do we support torch array with tensorflow backend ? Seems weird to me

Reply:
Yup xD, cause why not ... But yes, its not efficient at allll. There weirdly is no way to convert torch tensor to tensorflow tensor, without copying to CPU first... However, its just a way to prevent issues and bugs, it reallly isnt a usecase.

Reply:
well, okay, I'll let you merge :)